src/main/groovy/org/shipkit/gradle/ReleaseNeededTask.java

```diff
@@ -144,6 +147,26 @@ public ReleaseNeededTask setExplosive(boolean explosive) {
         return !releaseNotNeeded;
     }
 
+    private void logDifferencesBetweenPublications() {
+        if(publicationsComparators.isEmpty()){
+           return; // no differences to log
+        }
+        LOG.lifecycle("\n  Results of publications comparison:\n");
+        for(PublicationsComparatorTask task : publicationsComparators){
+            for(Diff diff : task.getDifferences()){
```

Comment:
At some point let's think how we can make comparator task incremental. For example, it can drop diff file. If the diff file is empty (or not present, it means there are no differences). Making every task incremental will be very powerful (instant builds) and will increase the credibility of Shipkit. Plus, it will be an exemplary implementation of Gradle plugins :D